app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -119,9 +121,21 @@ def instance_storage(persister_hardware, instance)
   def instance_operating_system(persister_instance, instance)
     image_id = instance&.dig(:image, :id)
     os = img_to_os[image_id] || pub_img_os(image_id)
+    os_name = normalize_os(os)
     persister.operating_systems.build(
       :vm_or_template => persister_instance,
-      :product_name   => os
+      :product_name   => os_name,
+      :version        => os
+    )
+  end
+
+  def image_operating_system(persister_image, image)
+    os_name = image&.dig(:operating_system, :name)
+    normalized_os_name = normalize_os(image&.dig(:operating_system, :name))
```

Comment:
Looks like you can just pass `os_name` in here instead of the `image&.dig` twice